js/tabber.js

```diff
@@ -35,7 +35,11 @@
 			nav.on('click', 'a', function(e) {
 				var title = $(this).attr('title');
 				e.preventDefault();
-				location.hash = '#' + title;
+				if ( history.pushState ) {
+					history.pushState( null, null, '#' + title );
```

Comment:
This has the same effect as location.hash however it doesn't actually visit the hash.

If there is a tab and a heading with the same name, the tab will open properly but the hash will jump the page to its respectively anchor tag.

This doesn't fix it but it's a small patchwork change that kinda sorta mitigates it when changing tabs.